src/module.c

```diff
@@ -5746,6 +5749,10 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
  *              invoking the command, the error is returned using errno mechanism.
  *              This flag allows to get the error also as an error CallReply with
  *              relevant error message.
+ *     * 'D' -- A "Dry Run" mode.  Return before executing the underlying call().
+ *              if everything succeeded, it will return with a NULL, otherwise it will
+ *              return with a CallReply object denoting the error, as if it was called with
```

Comment:
any reason not to explicitly require `E`?
maybe the error code mode would be useful for someone without the need to slow things down with error strings.

Reply:
I'm not sure how to indicate an error or not.  i.e. we can always return NULL and require user to check errno, but that seems error prone.  It seemed easier to indicate no error as NULL and error as the message / reason why.  But I'm open to discussion.  I do think dry run has to have a mode where it returns the error string, as very likely would be returned to the user.

Reply:
ok